iotdb-core/datanode/src/main/java/org/apache/iotdb/db/storageengine/StorageEngine.java

```diff
@@ -852,6 +853,8 @@ public TSStatus writeLoadTsFileNode(
       return status;
     }
 
+    LoadTsFileRateLimiter.getInstance().acquire(pieceNode.getDataSize());
```

Comment:
pieceNode.getDataSize() always return 0 here. it's fake on RPC receiver.

Reply:
Really BAD design...